README.md

```diff
@@ -30,7 +30,7 @@
 ```bash
 git clone https://github.com/OpenAccess-AI-Collective/axolotl
 
-pip3 install -e .[int4]
+pip3 install -e .[gptq]
```

Comment:
I think default in QuickStart should not be gptq if you recommend the `-e .` below.